packages/app/styles/LogSidePanel.module.scss

```diff
@@ -13,6 +13,13 @@ $padding-x: 24px;
   border-left: 1px solid $slate-900;
 }
 
+.panelDragBar {
```

Comment:
nit on 2 space indentation :)

minor UI improvement, I think it'd be nice if the bar lit up a bit when hovered to make it more clear it's interactive as well:

```scss
.panelDragBar {
  position: absolute;
  width: 3px;
  height: 100%;
  cursor: col-resize;
  background-color: transparent;

  &:hover {
    background-color: $slate-900;
    transition: background-color 0.2s ease-in-out;
  }
}
```